pupil/public/javascripts/mtc-check.js

```diff
@@ -1,25 +1,24 @@
-/* global Touch DocumentTouch Event */
-
 (function (global) {
   'use strict'
-  var GOVUK = global.GOVUK || {}
-  var startTime
-  var pageInterval = 5  // seconds allowed per page
-  var preloadInterval = 2 // seconds for student preparation
-  var deadline
-
-  var registerInputs = []
-  var isTouch = 'ontouchstart' in window || navigator.msMaxTouchPoints || navigator.maxTouchPoints ||
+
+  const GOVUK = global.GOVUK || {}
```

Comment:
Only way to use this would be with a transpiler like babel but that's part of future conversation